dev/AppNotifications/ShellLocalization.cpp

```diff
@@ -246,7 +247,9 @@ HRESULT Microsoft::Windows::AppNotifications::ShellLocalization::DeleteIconFromC
     std::wstring notificationAppId{ Microsoft::Windows::AppNotifications::Helpers::RetrieveNotificationAppId() };
 
     // path: C:\Users\<currentUser>\AppData\Local\Microsoft\WindowsAppSDK\{AppGUID}.png
-    std::path iconFilePath{ RetrieveLocalFolderPath() / (notificationAppId + c_pngExtension) };
+    std::path iconFilePath{ RetrieveLocalFolderPath() };
```

Comment:
Why break this up?

Reply:
I thought to follow some guidance from Howard about using std::filesystem::path, but it was already there. Will revert.

Reply:
Using std::path was the guidance and nothing changes there. We can compress all this to a single line though instead of making it so verbose.